docs/usage.txt

```diff
@@ -131,7 +131,7 @@ exception. If DEBUG is ``False`` these files will be silently stripped.
     Django's `caching documentation`_).
 
 .. _memcached: http://memcached.org/
-.. _caching documentation: https://docs.djangoproject.com/en/2.1/topics/cache/#memcached
+.. _caching documentation: https://docs.djangoproject.com/en/dev/topics/cache/#memcached
```

Comment:
Looks like /stable/ exists and redirects, it should be a better target.

```suggestion
.. _caching documentation: https://docs.djangoproject.com/en/stable/topics/cache/#memcached
```

Reply:
agreed, I was unaware of the existence of /stable/, in which case the link to template loaders on the same page (also points to dev) should be updated too.